### DISCUSSION TITLE
Move this to homeassistant core

### DISCUSSION THREAD

**Discussion by rokam** [CyrielRct/midea_ac_lan#4] [Ideas]
We'll be able to move this to [core](https://github.com/home-assistant/core), but we need to do:
- [x] separate the lib to access the devices
- [ ] add unittest to the component

I did create a repo for the first item: https://github.com/rokam/midea-local . I still need to create a release and add some tests.

**Comment by attilaersek** [comment]
Let me know if you need help in the development!
